src/nativeops.jl

```diff
@@ -0,0 +1,336 @@
+#=doc
+Native operators in Julia
+=========================
+=#
+module Native
+import ..mx
+
+#=doc
+.. class:: Operator
+
+   Abstract type user have to derive from to implement Operators in Julia
+=#
+abstract Operator
+
+#=doc
+.. function:: forward(op :: Operator, in_data, out_data)
+=#
+function forward(:: Operator, in_data, out_data)
+  out_data[0][:] = in_data[0]
```

Comment:
Copied that over from python by mistake. I decided instead of defining default function to throw a method error here.